marathon-gradle-plugin/src/main/kotlin/com/malinskiy/marathon/gradle/DistributionInstaller.kt

```diff
@@ -9,16 +9,15 @@ import java.nio.file.attribute.PosixFilePermission
 import java.util.zip.ZipFile
 
 class DistributionInstaller {
-    fun install(marathonBuildDir: File): File {
+    fun install(marathonBuildDir: File): Pair<File, File> {
```

Comment:
Looks like we never use result of this function

Reply:
You're correct, but this is mostly because I don't know how to propagate these between tasks except just with a convention